sdk/core/azure-core/build.gradle

```diff
@@ -34,5 +34,7 @@ android {
 
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
+    implementation 'androidx.annotation:annotation:1.1.0'
```

Comment:
Same comment here as in 139 about the dependency version. Also we should be consistent about double vs single quotes